### PR TITLE
Restaurant List Page

### PR DESCRIPTION
**Pull request by zhangwy324**
### Completed `/restaurants` page. 

This page includes:

- Title
- A card for each restaurant
   - name
   - description
   - category
   - price (amount of dollar signs)
   - Opening hours of current weekday
- Each card is Linked to the corresponding `/restaurants/[id]` page

### CODE REVIEW

File: components/Restaurant_card.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import Restaurant from "@/models/Restaurant";
+import Link from "next/link";
+
+interface ResCardProps {
+  restaurant: Restaurant;
+}
+
+export default function Restaurant_card({ restaurant }: ResCardProps) {
```
Comment (ajaygandecha) [inline components/Restaurant_card.tsx]: Nit: Feel free to refactor to be `RestaurantCard` for convention sake!

**Review by ajaygandecha — CHANGES_REQUESTED** [review]
Looks great, amazing work!! Really like the features and design. One small nitpick comment but once that is fixed, this will be ready to merge!

File: components/Restaurant_card.tsx (hunk 1 above)
Comment (zhangwy324) [reply components/Restaurant_card.tsx]: Fixed!

**Comment by ajaygandecha** [comment]
Sorry @zhangwy324, the name of the component has to be edited, not the props input!

**Review by ajaygandecha — APPROVED** [review]
LGTM! Ready to merge - great work

**Pull request merged**